src/spring/azext_spring/commands.py

```diff
@@ -486,5 +492,26 @@ def load_command_table(self, _):
                             exception_handler=handle_asc_exception) as g:
         g.custom_command('list', 'managed_component_instance_list', validator=validate_instance_list)
 
+    with self.command_group('spring job', custom_command_type=job_cmd_group,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_command('create', 'job_create', is_preview=True, validator=job_validators.validate_job_create)
+        g.custom_command('update', 'job_update', is_preview=True, validator=job_validators.validate_job_update)
+        g.custom_command('delete', 'job_delete', is_preview=True, validator=job_validators.validate_job_delete)
+        g.custom_show_command('show', 'job_get', supports_no_wait=True, is_preview=True, validator=job_validators.validate_job_get)
+        g.custom_command('list', 'job_list', supports_no_wait=True, is_preview=True, validator=job_validators.validate_job_list)
+        g.custom_command('deploy', 'job_deploy', supports_no_wait=True, is_preview=True, validator=job_validators.validate_job_deploy)
+        g.custom_command('start', 'job_start', supports_no_wait=True, is_preview=True, validator=job_validators.validate_job_start)
```

Comment:
> is_preview=True

Could we consider adding this flag to the command group level instead of adding it to every command

Reply:
Thanks. Updated.